concepts/match-basics/about.md

```diff
@@ -3,23 +3,31 @@
 The `match` expression allows for easy comparison of a value against a series of patterns and afterwards running the code based on the matched pattern. Literal values, variable names, wildcards, and many other things are what makes up a pattern.
 
 ```rust
+#[derive(Drop)]
 enum Color {
     Red,
     Yellow,
     Green,
 }
 
-fn print_color_type(color: Color) {
+fn specified_color(color: Color) {
     match color {
         Color::Red => println!("The color is Red"),
         Color::Yellow => println!("The color is Yellow"),
         Color::Green => println!("The color is Green"),
     }
 }
 
-// To handle multiple lines of code, the `match` arm of code should be wrapped in curly braces `{}`:
+fn main() {
+    let color = Color::Yellow;
+    specified_color(color);
+}
+```
 
-fn print_color_name(color: Color) -> felt252 {
+To handle multiple lines of code, the `match` arm of code should be wrapped in curly braces `{}`:
+
+```rust
+fn specified_color(color: Color) -> felt252 {
     match color {
         Color::Red => {
             println!("The color is Red");
```

Comment:
Update the function code to match [`specified_color` from SBE](https://starknet-by-example.voyager.online/getting-started/cairo_cheatsheet/match.html)

Reply:
I'd be adding the SBE link to the link json since this example is extracted from there.

Reply:
Not necessary, the _links.json_ file should include links to resources where students can learn more about the subject, and SBE contains no information that is not contained in The Cairo Book.